test/e2e/helpers.go

```diff
@@ -118,16 +118,16 @@ func requiredTemplateFunc(warn string, input interface{}) (interface{}, error) {
 	return input, nil
 }
 
-func makeBin(args ...string) *testutil.Exec {
+func makeBin(args ...string) *exec.Exec {
 	return makeBinWithPath(filepath.Clean("../../"), args...)
 }
 
-func makeBinWithPath(path string, args ...string) *testutil.Exec {
-	return testutil.NewExec("make",
-		testutil.WithArgs(args...),
-		testutil.WithEnv(os.Environ()),
-		testutil.InDir(path),
-		testutil.StdoutDebug,
+func makeBinWithPath(path string, args ...string) *exec.Exec {
+	return exec.NewExec("make",
```

Comment:
I'm really not fun of the name, in code it's confusing with `os/exec`. Let's imagine something like, `testexec`, `testbin` or something similar.

Reply:
That's a good point, I'll update the PR.

Reply:
@kron4eg Renamed to `testexec`, PTAL if it's okay now.